app/models/course.rb

```diff
@@ -1,4 +1,6 @@
 class Course < ApplicationRecord
+  validates :name, presence: true
```

Comment:
I don't mind adding this however, I think since we've not done messages for the others maybe we add them at the time we build the tooling to avoid inconsistencies